### PR TITLE
Add LoadBalance Plugin

### PR DESCRIPTION
**Pull request by LinuxSuRen**
Hosting Ticket : https://issues.jenkins-ci.org/browse/HOSTING-510
Repo : https://github.com/jenkinsci/agent-loadbalance-plugin

### CODE REVIEW

File: permissions/plugin-agent-loadbalance.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "loadbalance"
```
Comment (daniel-beck) [inline permissions/plugin-agent-loadbalance.yml]: Wrong name.

**Comment by oleg-nenashev** [comment]
As a maintainer of Job Restrictions Plugin, I am against the current implementation: https://github.com/jenkinsci/agent-loadbalance-plugin/blob/master/src/main/java/com/surenpi/jenkins/loadbalance/LoadBalanceRestriction.java . It is a hack, LoadBalancer should be used instead as we discussed in the hosting request. 

I do not block hosting of the plugin in the current state, but I will reject to triage any Job Restriction issues if this plugin is installed.

File: permissions/plugin-agent-loadbalance.yml (hunk 1 above)
Comment (LinuxSuRen) [reply permissions/plugin-agent-loadbalance.yml]: Already fixed. Thanks.

**Comment by LinuxSuRen** [comment]
@oleg-nenashev Totally understand. But in that situation, it must use restrction not balancer. Maybe I should make a change, 'ResourceRestriction' could be better than 'LoadBalanceRestriction'.

**Pull request merged**